firmware/mcal/cli/periph/analog_output.h

```diff
@@ -0,0 +1,29 @@
+/// @author Tamara Xu
+/// @date 2024-10-03
+
+#pragma once
+
+#include <iostream>
+#include <string>
+
+#include "shared/periph/analog_output.h"
+
+namespace mcal::cli::periph {
+
+class AnalogOutput : public shared::periph::AnalogOutput {
+public:
+    AnalogOutput(std::string name) : name_(name) {}
+
+    void Set(float voltage) {
+        voltage_ = voltage;
```

Comment:
do we need to store `voltage_` in the object? I think we can just use the argument in the print statement and forget about it